great_expectations/rule_based_profiler/profiler.py

```diff
@@ -111,88 +131,33 @@ def __init__(
     def profile(
         self,
         *,
-        validator=None,
-        batch=None,
-        batches=None,
-        batch_request=None,
-        batch_ids=None,
-        data_context=None,
-        expectation_suite_name=None,
```

Comment:
To confirm - the reason for this change is to better match the existing UserConfigurableProfiler, where the profiler is instantiated with a config, and that config is not changed when profiling?

Reply:
@talagluck Correct.